### ISSUE TITLE
Feature Request : Have a favorite menu to save its own favorite games

### ISSUE THREAD

**Issue by PaulLouviers** [a2-4am/4cade#563]
Now we have the chance to near 500 hundred games, it is really painful to keep in mind which games are good or not. On my side I have a small book to remind me, which game is my TOP 20.

Feasible ? thanks

**Comment by a2-4am** [comment]
Yes this is a good idea. Realistically we're not going to tackle it in v5, but I will leave this issue open.

**Comment by wyatt-wong** [comment]
> Now we have the chance to near 500 hundred games, it is really painful to keep in mind which games are good or not. On my side I have a small book to remind me, which game is my TOP 20.
> 
> Feasible ? thanks

The latest

> Now we have the chance to near 500 hundred games, it is really painful to keep in mind which games are good or not. On my side I have a small book to remind me, which game is my TOP 20.
> 
> Feasible ? thanks

The latest build 4113 (as at Jan 14 2025) had 505 games.
